From 21012a8ec368d3e88a5f3d00030d061e21fcc9a7 Mon Sep 17 00:00:00 2001
From: Gregory Hainaut <gregory.hainaut@gmail.com>
Date: Fri, 2 Sep 2016 22:53:52 +0200
Subject: [PATCH] 1.4.x: x86emitter: fix compilation issue with recent GCC

close #1544
---
 common/src/x86emitter/cpudetect.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/common/src/x86emitter/cpudetect.cpp b/common/src/x86emitter/cpudetect.cpp
index 717ca8972a..dc184fed9a 100644
--- a/common/src/x86emitter/cpudetect.cpp
+++ b/common/src/x86emitter/cpudetect.cpp
@@ -16,6 +16,9 @@
 #include "PrecompiledHeader.h"
 #include "cpudetect_internal.h"
 #include "internal.h"
+#ifdef __linux__
+#include "fxsrintrin.h"
+#endif
 
 using namespace x86Emitter;
 
